src/showingpreviously/cinemas/centre_for_the_moving_image.py

```diff
@@ -0,0 +1,134 @@
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+import showingpreviously.requests as requests
+
+import datetime
+from bs4 import BeautifulSoup
+from typing import Iterator, Tuple
+
+
+DAYS_PREVIOUS = 3
+
+
+CHAIN = Chain(name='Centre for the Moving Image')
+EDINBURGH_CINEMA = Cinema(name='Filmhouse Edinburgh', timezone='Europe/London')
+BELMONT_CINEMA = Cinema(name='Belmont Filmhouse', timezone='Europe/London')
+
+
+KNOWN_SHOWING_TYPE_ATTRIBUTES = [
+    'audio-described',
+    'subtitled',
+    'captioned'
+]
+
+
+CINEMAS_LIST = [
+    {'base_url': 'https://www.filmhousecinema.com', 'cinema': EDINBURGH_CINEMA},
+    {'base_url': 'https://www.belmontfilmhouse.com', 'cinema': BELMONT_CINEMA},
+]
+
+
+film_page_details_cache = {}
+
+
+def get_response(url: str) -> requests.Response:
+    r = requests.get(url)
+    if r.status_code != 200:
+        raise CinemaArchiverException(f'Got status code {r.status_code} when fetching URL {url}')
+    return r
+
+
+def get_days_to_fetch() -> [datetime.datetime]:
+    # Fetch films from the range of (today - DAYS_PREVIOUS) to today
+    base = datetime.date.today()
+    date_list = [base - datetime.timedelta(days=x) for x in range(DAYS_PREVIOUS)]
+    return date_list
+
+
+def get_showings_for_date(cinema: dict[str, any], fetch_date: datetime.datetime) -> [Showing]:
```

Comment:
Incorrect type here - `fetch_date` is actually `datetime.date` not `datetime.datetime`.